_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -10614,6 +10614,16 @@
 	},
 /turf/open/floor/iron/dark,
 /area/security/detectives_office)
+"bJt" = (
+/obj/structure/cable,
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/obj/effect/spawner/random/engineering/tracking_beacon{
+	spawn_loot_chance = 35
```

Comment:
This should probably be a subtype, rather than manually editing `spawn_loot_chance = 35` for a million different instances

Reply:
True.  I will edit the chance to 35 for the base class and remove the var edits.

Reply:
Don't edit the chance to 35 for the base class, make a subtype. The beacon at arrivals is guaranteed, and that chance shouldn't have to be var-edited in on the maps.

Reply:
The beacon in arrivals isn’t a random spawner.  All the other beacons are random spawners with a spawn chance.

Reply:
I see. Then yeah, as long as the random spawner isn't used anywhere else your implementation should be fine, sorry!

Reply:
This was implemented with 30e55a54ef06714ff7c8d4b77366fd68ca2e54e0